modules/ztp-factory-provisioning-hub-prereqs.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="factory-installing-edge-sites_{context}"]
+= Hub node prerequisites
+
+This document assumes an external hub cluster, which runs an OpenShift deployment (Compact), RHACM-based cluster with assisted service installed.
```

Comment:
Do we need to call out OCP 4.9 or higher, and RHACM 2.4 or higher?

Reply:
I have separate PR for 4.9 https://github.com/openshift/openshift-docs/pull/44393

Reply:
I have separate PR for 4.9 as some diffs in spokes.yaml file https://github.com/openshift/openshift-docs/pull/44393